Sources/TarkaUI/Components/ToggleRow/TUIToggleRow.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  TUIToggleRow.swift
+//
+//
+//  Created by Arvindh Sukumar on 21/02/24.
+//
+
+import SwiftUI
+
+/// A row with a content view and a toggle switch
+/// - Parameters:
+/// - contentView: The content view to be displayed
+/// - toggle: The toggle switch to be displayed
+public struct TUIToggleRow<Content: View>: View {
+  public var contentView: () -> Content
+  public var toggle: () -> TUIToggleSwitch
+  
+  public var body: some View {
+    HStack(spacing: Spacing.baseHorizontal) {
+      contentView()
+      Spacer()
```

Comment:
```suggestion
   .frame(maxWidth: .infinity, alignment: .leading)
```

Reply:
Spacer() is needed to ensure that the toggle view is right-aligned